apps/emails-and-messages/src/modules/smtp/ui/sender-section.tsx

```diff
@@ -0,0 +1,79 @@
+import { SmtpConfiguration } from "../configuration/smtp-config-schema";
+import { BoxWithBorder } from "../../../components/box-with-border";
+import { Box, Button } from "@saleor/macaw-ui/next";
+import { defaultPadding } from "../../../components/ui-defaults";
+import { useDashboardNotification } from "@saleor/apps-shared";
+import { trpcClient } from "../../trpc/trpc-client";
+import { useForm } from "react-hook-form";
+import { BoxFooter } from "../../../components/box-footer";
+import { SectionWithDescription } from "../../../components/section-with-description";
+import {
+  SmtpUpdateSender,
+  smtpUpdateSenderSchema,
+} from "../configuration/smtp-config-input-schema";
+import { Input } from "../../../components/react-hook-form-macaw/Input";
+import { zodResolver } from "@hookform/resolvers/zod";
+import { setBackendErrors } from "../../../lib/set-backend-errors";
+
+interface SenderSectionProps {
+  configuration: SmtpConfiguration;
+}
+
+export const SenderSection = ({ configuration }: SenderSectionProps) => {
```

Comment:
I think it will be good for the maintenance if you name components like `SendgridSenderSection` and `SMTPSenderSection`. In JS we can't scope modules so we can easily import wrong one, because the names are the same

We can use some libraries that will build-time prohibit imports (https://github.com/smikula/good-fences)

So you should never be able to import anything from "smpt" in "sendgrid" and the opposite

Reply:
Names updated for duplicated components